deploy/charts/istio-csr/values.yaml

```diff
@@ -9,6 +9,9 @@ image:
   # -- Kubernetes imagePullPolicy on Deployment.
   pullPolicy: IfNotPresent
 
+  # Pull secret name for private repository hosting of istio-csr
```

Comment:
Please run [helm-docs](https://github.com/norwoodj/helm-docs/) to update the README.md.

Reply:
Waiting for this [PR](https://github.com/cert-manager/istio-csr/pull/150/files) so this is a bit clearer in future :slightly_smiling_face:

Reply:
Nope, the README.md is out of sink and needs to be made right- hence the other PR to run the test in CI. Don't worry about the rest for now, just make sure your changes are included in the README.md and we can release

Reply:
done